core/templates/dev/head/player/progress_dots_directive.html

```diff
@@ -80,25 +80,17 @@
     <li class="oppia-progress-arrow" ng-show="dots.length > 1"
         ng-class="{'oppia-progress-arrow-active' : currentDotIndex > 0}"
         ng-click="decrementCurrentDotIndex()">
-<<<<<<< HEAD
+
       <span class="sr-only" translate="I18N_PLAYER_BACK"></span>
-      <span class="glyphicon glyphicon-arrow-left" aria-hidden="true">
-=======
-      <span class="sr-only">Back</span>
       <i class="material-icons oppia-vcenter" aria-hidden="true">&#xE5C4;</i>
->>>>>>> develop
       </span>
     </li>
     <li ng-repeat="dot in dots track by $index"
         class="oppia-animate-progress-dot"
         ng-show="$index >= leftmostVisibleDotIndex && $index <= rightmostVisibleDotIndex">
       <span class="oppia-progress-dot oppia-progress-dot-active"
         ng-if="$index === currentDotIndex" ng-show="dots.length > 1"
-<<<<<<< HEAD
-        tooltip="<['I18N_PLAYER_CARD_NUMBER_TOOLTIP' | translate]><[$index +1]>" tooltip-placement="bottom"
-=======
-        tooltip="Card #<[$index +1]>" tooltip-placement="left"
->>>>>>> develop
+        tooltip="<['I18N_PLAYER_CARD_NUMBER_TOOLTIP' | translate]><[$index +1]>" tooltip-placement="left"
```

Comment:
Yes this needs further translation, but I didn't want to mix things in the commit. I added a TODO comment.